drivers/boston_project_scorecard/app/models/boston_project_scorecard/project_performance.rb

```diff
@@ -147,6 +147,20 @@ def materials_concern_score
 
         materials_concern
       end
+
+      def returns_to_homelessness_value
+        percentage_string(returns_to_homelessness)
+      end
+
+      def returns_to_homelessness_score
+        return 0 unless returns_to_homelessness.present?
+
+        return 12 if returns_to_homelessness.round <= 5
+        return 6 if returns_to_homelessness.round <= 25
+        return 4 if returns_to_homelessness.round <= 50
+
+        0
+      end
```

Comment:
Are we sure `round` is correct? So if returns_to_homelessness of 5.4 we'd get a score 6, not 12. Is that intended?